### ISSUE TITLE
threejs rev 0.126.x - Face3 removed

### ISSUE THREAD

**Issue by RealTecWare** [gkjohnson/three-mesh-bvh#189]
Hy, 

today I updated my project to threejs rev 0.125.x and noticed an Error in "ThreeIntersectionUtilities.js".

The problem was that there is the Face3 class in use, what is not supported in threejs any more .

Error Line:
https://github.com/gkjohnson/three-mesh-bvh/blob/3369022cb6bc43a66c23fb28b45529c92610e524/src/Utils/ThreeIntersectionUtilities.js#L67

You could fix this easy by change it to following:

```javascript
const face = {
  a: a,
  b: a,
  c: c,
  normal: new Vector3(),
  materialIndex: 0
};


Triangle.getNormal( vA, vB, vC, face.normal );

intersection.face = face;
```

This is a copy of the new source from Mesh.js:

https://github.com/mrdoob/three.js/blob/350f0a021943d6fa1d039a7c14c303653daa463f/src/objects/Mesh.js#L403

**Comment by gkjohnson** [comment]
Thanks for the report! Your change looks good. Would you like to make a PR with adjustment?

**Comment by gkjohnson** [comment]
Fixed with #190

**Comment by gkjohnson** [comment]
It looks like this was actually removed in v126. I'll publish a new revision at the end of month.
